tc/src/main/webapp/WEB-INF/templates/courseDetails.html

```diff
@@ -14,18 +14,46 @@
             </header>
             <div class="form-horizontal">
                 <div class="control-group">
-                    <td th:text="${course.name}">Name </td>
+                    <div class="control-label">Name</div>
+                    <div id="titleField" class="controls text" th:text="${course.name}"></div>
                 </div>
                 <div class="control-group">
-                    <td th:text="${course.description}">Description </td>
+                    <div class="control-label">Owner</div>
+                    <div id="linksField" class="controls text" th:text="${course.owner.email}"></div>
                 </div>
                 <div class="control-group">
-                    <td th:text="${course.links}">Links </td>
+                    <div class="control-label">Description</div>
+                    <div id="linksField" class="controls text" th:text="${course.description}"></div>
                 </div>
-                <div class="form-actions">
-                    <a class="btn" href="courses">Back</a>
+                <div class="control-group">
+                    <div class="control-label">Links</div>
+                    <div id="linksField" class="controls text" th:text="${course.links}"></div>
+                </div>
+                <div class="control-group" th:if="${course.getSubscribers().size() &gt; 0}">
+                    <div class="control-label">Subscribers</div>
+                    <div id="subscribersField" class="controls text"> 
+                        <a th:href="@{'/courses/'+${course.getId()}+ '/participants'}"
+                           th:text="${course.getSubscribers().size()}">
+                        </a>
+                    </div>
+                </div>
+                <div class="control-group" th:if="${course.getAttenders().size() &gt; 0}">
```

Comment:
I believe that you you can access these data like it is stored in public fields: course.attenders instead of course.getAttenders()